frontend/src/pages/index.tsx

```diff
@@ -1,7 +1,55 @@
+import { FieldValues, SubmitHandler, useForm } from "react-hook-form";
+
+import { Checkbox } from "../components/Checkbox";
+import Radio from "../components/Radio";
+import { Textfield } from "../components/Textfield";
+
 export default function Home() {
+  const dietaryList = ["Nuts", "Eggs", "Seafood", "Pollen", "Dairy", "Other"];
```

Comment:
Lets remove these components from the index page before merging since we're not going to need it here.